### PR TITLE
Addingswig

### PR DESCRIPTION
**Pull request by tasyne**
Hi. Here's the pull request for yet another attempt to get the old Kovan library to compile on a Wallaby. The camera module is disabled (as much as possible) to get even basic sensor and motor functionality working with Python would be a major success.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -74,26 +77,52 @@ elseif(WIN32)
   set(ZBAR_DIR "C:/Program Files (x86)/ZBar")
 endif()
 
-find_package( OpenCV REQUIRED )
-find_package(JPEG REQUIRED)
-find_package (Threads)
+#find_package( OpenCV REQUIRED )
+#find_package(JPEG REQUIRED)
+#find_package (Threads)
```
Comment (jsoutherland) [inline CMakeLists.txt]: Did you have trouble with the Threads package?  This is how we find pthread.

**Comment by jsoutherland** [comment]
Thanks @tasyne this is my priority today.  I'm getting a VM set up real fast to try it from scratch.

**Comment by tasyne** [comment]
The removal of Threads was likely a mistake as I was wrapping things up.
the cmake/OpenCVConfig.cmake file (that's probably in the wrong directory)
was also the remnants of a past attempt (a longshot that didn't pan out)

**Comment by jsoutherland** [comment]
@tasyne I got this into #65  for testing along with changes from my fork.

**Pull request closed without merging**